test/xebow_test.exs

```diff
@@ -1,7 +1,9 @@
 defmodule XebowTest do
   use ExUnit.Case
 
-  test "has layout" do
-    assert %Layout{} = Xebow.layout()
+  describe "Xebow Application" do
+    test "has layout" do
+      assert %Layout{} = Xebow.layout()
+    end
```

Comment:
:information_source: (optional)

I would leave this code alone in this PR since it's out of scope and change it in part of the application refactoring work.

Reply:
Good point.